xml/vt_installation.xml

```diff
@@ -258,4 +258,75 @@
    />.
   </para>
  </sect1>
+ <sect1 xml:id="sec-vt-installation-nested-vms">
+  <title>Enable Support for Nested Virtualization in &kvm;</title>
+  <important>
+   <title>Technology Preview</title>
+   <para>
+    &kvm;'s nested virtualization is still a technology preview.  It is
+    provided for testing purposes and is not supported.
+   </para>
+  </important>
+  <para>
+   Nested guests are &kvm; guests run in a &kvm; guest. When describing nested
+   guests, we will use the following virtualization layers:
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>L0</term>
+    <listitem>
+     <para>
+      A bare metal host running &kvm;.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L1</term>
+    <listitem>
+     <para>
+      A virtual machine running on L0. Because it can run another &kvm;, it is
+      called a <emphasis>guest hypervisor</emphasis>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L2</term>
+    <listitem>
+     <para>
+      A virtual machine running on L1. It is called a <emphasis>nested
+       guest</emphasis>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   Nested virtualization is useful mainly in cloud environments as you can
+   manage your own virtual machines there directly with your favorite
+   hypervisor of choice. It can also enable the live migration of hypervisors
+   and their guest virtual machines as a single entity.
+ </para>
+ <para>
+  To enable nesting, set the <option>nested</option> module parameter to
+  <literal>Y</literal>.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    For Intel CPUs, edit <filename>/etc/modprobe.d/kvm_intel.conf</filename>
+    and add the following line:
+   </para>
+   <screen>options kvm-intel nested=Y</screen>
+  </listitem><listitem>
+   <para>
+    For AMD CPUs, edit <filename>/etc/modprobe.d/kvm_amd.conf</filename>
+    and add the following line:
+   </para>
+   <screen>options kvm-amd nested=Y</screen>
+  </listitem>
+ </itemizedlist>
+ <para>
+  When your L0 host is capable of nesting, you will be able to start an L1
+  guest with the <option>-cpu host</option> option.
```

Comment:
... guest with the <option>-cpu host</option> QEMU command-line option, or by adding the vmx (for Intel CPUs) or the svm  (for AMD CPUs) cpu feature to the -cpu <model> QEMU command-line option.

(or words to that effect).